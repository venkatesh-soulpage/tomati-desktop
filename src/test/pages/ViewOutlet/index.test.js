import React from "react";
import { render } from "@testing-library/react";
import Index from "pages/ViewOutlet";
import { Router } from "react-router-dom";
import history from "utils/history";
import { Provider } from "react-redux";
import configureStore from "store";

describe("ViewOutlet page", () => {
  it("matches snapshot", () => {
    const store = configureStore();
    const { asFragment } = render(
      <Provider store={store}>
        <Router history={history}>
          <Index />
        </Router>
      </Provider>,
      {}
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
