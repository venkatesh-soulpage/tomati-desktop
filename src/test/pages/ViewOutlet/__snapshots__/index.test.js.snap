// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ViewOutlet page matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="p-3"
  >
    <div
      class="border background"
      style="height: 244px; background: url(undefined);"
    >
      <div
        class="row h-100 justify-content-center blur"
      >
        <div
          class="col-md-3 d-flex justify-content-center align-self-center"
        >
          <div
            class="rounded-circle logo-img1 ml-4"
          >
            <img
              alt="logo"
              class=" border1-white img-fluid h-100 rounded-circle"
              height="120px"
              width="120px"
            />
            <label
              for="logoImage"
            >
              <img
                alt="camera"
                class="cr-p text-white ml-64"
                src="CameraIcon.svg"
              />
            </label>
          </div>
          <div
            class="form-group"
          >
            <input
              class="d-none form-control-file"
              id="logoImage"
              type="file"
            />
          </div>
        </div>
        <div
          class="col-md-9 align-self-center"
        >
          <h4
            class="text-white font-weight-bold fs-30 backdrop p-1"
          />
          <p
            class="text-white font-weight-light fs-12 w-58 backdrop p-1"
          />
        </div>
        <div
          class="form-group"
        >
          <input
            class="d-none form-control-file"
            id="coverImage"
            type="file"
          />
        </div>
        <div
          class="col-11 d-flex pr-0 justify-content-end align-items-center"
        >
          <button
            class="btn p-2 btn-outline-dark text-white d-flex align-items-center cr-p b1-white mb-3 btn-backdrop "
          >
            <label
              class="p-0 m-0 cr-p"
              for="coverImage"
            >
              <svg
                class="fs-20"
                fill="currentColor"
                height="1em"
                viewBox="0 0 16 16"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M10.5 8.5a2.5 2.5 0 1 1-5 0 2.5 2.5 0 0 1 5 0z"
                />
                <path
                  d="M2 4a2 2 0 0 0-2 2v6a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V6a2 2 0 0 0-2-2h-1.172a2 2 0 0 1-1.414-.586l-.828-.828A2 2 0 0 0 9.172 2H6.828a2 2 0 0 0-1.414.586l-.828.828A2 2 0 0 1 3.172 4H2zm.5 2a.5.5 0 1 1 0-1 .5.5 0 0 1 0 1zm9 2.5a3.5 3.5 0 1 1-7 0 3.5 3.5 0 0 1 7 0z"
                />
              </svg>
              <span
                class="ml-2"
              >
                Edit Cover Photo
              </span>
            </label>
          </button>
        </div>
      </div>
      <div
        class="mt-3"
      >
        <div
          class="card bg-white border p-3"
        >
          <div
            class="d-flex align-items-center"
          >
            <div
              class=""
            >
              <a
                href="/dashboard/viewoutlet"
                style="color: rgb(44, 58, 86); text-decoration: none;"
              >
                <h6
                  class="m-0"
                >
                   
                  <img
                    alt="qr"
                    class="mr-2"
                    src="IconQR.svg"
                  />
                  QR Code
                </h6>
              </a>
            </div>
            <div
              class="mr-auto ml-5"
            >
              <a
                href="/dashboard/viewoutlet/about"
                style="color: rgb(44, 58, 86); text-decoration: none;"
              >
                <h6
                  class="m-0"
                >
                  About
                </h6>
              </a>
            </div>
            <div
              class="mr-auto ml-3"
            >
              <a
                href="/dashboard/viewoutlet/collaborators"
                style="color: rgb(44, 58, 86); text-decoration: none;"
              >
                <h6
                  class="m-0"
                >
                  Collaborators
                </h6>
              </a>
            </div>
            <div
              class="ml-auto mr-2"
            >
              <button
                class="btn btn-outline-dark rounded-pill"
              >
                Add Collaborators
              </button>
            </div>
            <div>
              <button
                class="btn btn-danger rounded-pill"
              >
                Update Menu
              </button>
            </div>
          </div>
        </div>
        <div
          class="card bg-white border p-5 mt-2"
        >
          <div
            class="d-flex align-items-top"
          >
            <div
              class="mr-5"
            >
              <img
                alt="menu-qr"
                class="border"
                height="204px"
                id="menu-qr"
                width="204px"
              />
            </div>
            <div
              class="mr-auto"
            >
              <h4
                class="text-dark"
              />
              <p
                class="text-dark font-weight-light w-56"
              />
              <button
                class="btn btn-danger mr-3 mt-1 rounded-pill"
              >
                Download QR Code
              </button>
              <button
                class="btn btn-outline-dark mt-1 px-4 rounded-pill "
              >
                Order Menu tags
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
